docs/software/social.md

```diff
@@ -135,6 +135,13 @@ It allows you to create your own website.
     - Database Host = `localhost`
     - Table Prefix = `wp_`
 
+    !!! note "Actively fill all fields!"
```

Comment:
Isn't `note` adding this pencil icon? I think the exclamation mark of `info` fits better here.

Reply:
`info` gives an 'i', no '!'.
See https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types

Reply:
Ah right, however it is more an eye catcher, fitting better for info about solving potential problems. IMO.

Reply:
@MichaIng: In case of an eyecatcher, I propose this:

![image](https://user-images.githubusercontent.com/57687516/223526800-615c0f7f-d025-4ccb-9158-5e3026b84570.png)

Reply:
This is the other extreme, going into "danger" direction. IMO the "info" is the right thing in the middle.

Reply:
I throwed a coin and you won for this time. :-)

Reply:
Maybe we should declare some standard regarding these boxes:
- "note" for info which can be interesting or an extension, but doesn't affect whether something succeeds or not
- "tip" for similar info, but such we want to further highlight, because they are pretty cool or contributed by others
- "info" for cases where one/something may not proceed/succeed when not taking into account
- "warning" for cases where one might damage or loose something when not taking into account
- There is also the "danger" box in red, which we do not currently use, IMO it isn't more highlighting than the warning, and not sure how to best differentiate between "warning" and "danger" boxes then.